ctx.go

```diff
@@ -1017,6 +1017,22 @@ func (c *Ctx) Render(name string, bind interface{}, layouts ...string) error {
 	buf := bytebufferpool.Get()
 	defer bytebufferpool.Put(buf)
 
+	// Check if the PassLocalsToViews option is enabled (By default it is disabled)
+	if c.app.config.PassLocalsToViews {
+		// Safely cast the bind interface to a map
+		bindMap, ok := bind.(Map)
+		// Check if the bind is a map
+		if ok {
+			// Loop through each local and set it in the map
+			c.fasthttp.VisitUserValues(func(key []byte, val interface{}) {
+				bindMap[string(key)] = val
```

Comment:
the map that is being handed in should only be expanded, not completely replaced

furthermore the values which are given directly to the render function should have more value

i.e. a value which is in the locals with key "abc" and one which is directly passed to the render function, the one which is passed to the render function should win

Reply:
@kingdevnl